docs/runtime/management.adoc

```diff
@@ -647,6 +647,46 @@ image:xp-760.svg[XP 7.6.0,opts=inline] Stops an application with specific applic
 ----
 No response body
 ----
+
+=== Subscribe to Server-Sent Events
+
+image:xp-760.svg[XP 7.6.0,opts=inline] The client initiates the SSE connection by using the media type text/event-stream in the Accept header. Then it gets updates automatically without requesting the server.
+
+.Url
+ GET app/events
+
+Currently that API supports the following events:
+
+- `list` - returns the details of applications which already installed
```

Comment:
Mention that this event always arrives first after subscription.

Reply:
I wrote about it below. https://github.com/enonic/doc-xp/pull/188/files#diff-4af2db204c1824abaebf26de983416440bbcd3c825de5450ea56ef0d0be2ec25R665